crates/paralegal-policy/src/context.rs

```diff
@@ -299,6 +299,8 @@ impl Context {
     /// Returns whether a node flows to a node through the configured edge type.
     ///
     /// Nodes do not flow to themselves. CallArgument nodes do flow to their respective CallSites.
+    ///
+    /// If you use flows_to with [`EdgeType::Control`], you might want to consider using has_ctrl_influence.
```

Comment:
Please explain briefly why someone would want to use `has_ctrl_influence` instead (what is the difference between that and using `EdgeType::Control` on this function).

Also please use a reference to `has_ctrl_influence` the same way we reference `EdgeType::Control`.

Reply:
done